datasets/ebi_chembl/pipelines/_images/run_csv_transform_kub/requirements.txt

```diff
@@ -0,0 +1,5 @@
+cloud-sql-python-connector
+google-cloud-storage
+pandas
+pg8000
```

Comment:
What is this?

Reply:
For function annotation I needed pg8000 module. Now as per latest changes to main(), removed pg8000 from requirements.txt